generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -128,7 +128,7 @@ public User createUser(ManagedUserDTO managedUserDTO) {
         user.setLastName(managedUserDTO.getLastName());
         user.setEmail(managedUserDTO.getEmail());
         if (managedUserDTO.getLangKey() == null) {
-            user.setLangKey("en"); // default language is English
+            user.setLangKey("<%= nativeLanguageShortName %>"); // default language is English
```

Comment:
Should probably modify the comment to say something like 
// defaults to native language